mob_test.go

```diff
@@ -76,70 +76,50 @@ func assertDetermineBranches(t *testing.T, branch string, qualifier string, bran
 	equals(t, expectedWip, wipBranch)
 }
 
-func TestEnvironmentVariables(t *testing.T) {
-	os.Setenv("MOB_REMOTE_NAME", "GITHUB")
-	defer os.Unsetenv("MOB_REMOTE_NAME")
-
-	os.Setenv("MOB_DEBUG", "true")
-	defer os.Unsetenv("MOB_DEBUG")
-
-	configuration = parseEnvironmentVariables(getDefaultConfiguration())
+func TestMobRemoteNameEnvironmentVariable(t *testing.T) {
+	configuration = setEnv("MOB_REMOTE_NAME", "GITHUB")
 
 	equals(t, "GITHUB", configuration.RemoteName)
-	equals(t, true, configuration.Debug)
 }
 
-func TestEnvironmentVariablesEmptyString(t *testing.T) {
-	os.Setenv("MOB_REMOTE_NAME", "")
-	defer os.Unsetenv("MOB_REMOTE_NAME")
-
-	configuration = parseEnvironmentVariables(getDefaultConfiguration())
+func TestMobRemoteNameEnvironmentVariableEmptyString(t *testing.T) {
+	configuration = setEnv("MOB_REMOTE_NAME", "")
 
 	equals(t, "origin", configuration.RemoteName)
 }
 
-func TestMobDoneSquashEnvironmentVariableDefault(t *testing.T) {
-	configuration = parseEnvironmentVariables(getDefaultConfiguration())
-
-	equals(t, true, configuration.MobDoneSquash)
-}
-
-func TestMobDoneSquashEnvironmentVariableEmpty(t *testing.T) {
-	// ASSUME default setting for mobDoneSquash is true
-	os.Setenv("MOB_DONE_SQUASH", "")
-	defer os.Unsetenv("MOB_DONE_SQUASH")
-
-	configuration = parseEnvironmentVariables(getDefaultConfiguration())
-
-	equals(t, true, configuration.MobDoneSquash)
+func TestBooleanEnvironmentVariables(t *testing.T) {
+	expectEnvironmentVariableSetsBooleanConfiguration(t, "MOB_DONE_SQUASH", true, func(configuration Configuration) interface{} { return configuration.MobDoneSquash })
```

Comment:
One can write such functions as part of getters for that struct. 

```golang
func (c Configuration) GetMobDoneSquash() bool {
    return c.MobDoneSquash
}
```

and with this function the test should become more readable, as we only need to pass in that function as a parameter.

Reply:
Here's the article from which I got my info: https://medium.com/rungo/anatomy-of-methods-in-go-f552aaa8ac4a